ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -264,7 +297,34 @@ public struct Avatar: View, TokenizedControlView {
                                         .frame(width: presenceIconFrameSideRelativeToOuterRing, height: presenceIconFrameSideRelativeToOuterRing,
                                                alignment: .bottomTrailing),
                                      alignment: .topLeading)
-                            .frame(width: overallFrameSide, height: overallFrameSide, alignment: .topLeading)
+                            .frame(width: presenceOverallFrameSide, height: presenceOverallFrameSide, alignment: .topLeading)
+                    })
+                    .modifyIf(shouldDisplayActivity, { thisView in
```

Comment:
The logic for presence and activity look pretty similar. Would it be possible to move them into a method, or something?